QModManager/Utility/Config.cs

```diff
@@ -0,0 +1,90 @@
+﻿namespace QModManager.Utility
+{
+    using Oculus.Newtonsoft.Json;
+    using System;
+    using System.Collections.Generic;
+    using System.IO;
+
+    internal static class Config
+    {
+        internal static bool CheckForUpdates
+        {
+            get => Get("Check for updates", true);
+            set => Set("Check for updates", value);
+        }
+
+        internal static bool EnableConsole
+        {
+            get => Get("Enable console", false);
+            set => Set("Enable console", value);
+        }
+
+        internal static bool EnableDebugLogs
+        {
+            get => Get("Enable debug logs", false);
+            set => Get("Enable debug logs", value);
+        }
+
+        internal static bool EnableDevMode
+        {
+            get => Get("Enable developer mode", false);
+            set => Set("Enable developer mode", value);
+        }
+
+        private static readonly string ConfigPath = Path.Combine(Environment.CurrentDirectory, "qmodmanager-config.json");
+
+        private static Dictionary<string, object> Cfg = new Dictionary<string, object>();
+        private static bool Loaded = false;
+
+        private static void Load()
+        {
+            if (!File.Exists(ConfigPath)) File.WriteAllText(ConfigPath, "{}");
+            string text = File.ReadAllText(ConfigPath);
+            Cfg = JsonConvert.DeserializeObject<Dictionary<string, object>>(text);
+        }
+
+        private static void Save()
+        {
+            string text = JsonConvert.SerializeObject(Cfg, Formatting.Indented);
+            File.WriteAllText(ConfigPath, text);
+        }
+
+        private static object Get(string field, object def = null)
```

Comment:
This `Get` method could be merged with the `Get<T>` method if the try-catch blocks are moved into the `Save()` and `Load()` methods directly.

Also, you don't need to put `Cfg.TryGetValue` inside of a `try` block. It is perfectly safe to call directly.